packages/third-party/tailwind-preset/src/core.ts

```diff
@@ -0,0 +1,304 @@
+// Copyright 2025 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+import * as pluginModules from 'plugins/lynx';
```

Comment:
Please use relative path instead.
```suggestion
import * as pluginModules from './plugins/lynx/index.js';
```

Reply:
Editied.